.eslintrc.json

```diff
@@ -63,6 +63,13 @@
       "rules": {
         "no-console": "off"
       }
+    },
+    {
+      "files": ["./**/*.spec.{ts,js}", "./cypress/**", "./demos/**", "./**/docs/**"],
+      "rules": {
+        "jsdoc/require-jsdoc": "off",
+        "@typescript-eslint/no-unused-vars": "off"
```

Comment:
It might be worth keeping this as `"warn"` or `"error"` for `/**/*.spec.{ts,js}`, to avoid bugs in unit test code.

Reply:
The current spec files are the main source of these warnings. I guess around 600+ warnings, which was polluting the output, so we weren't able to find the actual errors. 

So @knsv suggested we mute them for now and re-enable later once we fix the tests.

Reply:
Oh wow! That's a lot! Yep, that definitely makes sense to disable then.

If it's just a few, it's nice to leave it enabled on "warning", so that GitHub PRs will show that the warning was added, but with so many warnings, it makes `yarn lint` really hard to read.

Reply:
Yeah, at one point I was using `yarn lint | grep -v "warning"` to figure out what the errors were. 😂 
Once the diagram separation goes through, we can re-enable these.